### PR TITLE
Docs/WordPress.WhiteSpace.DisallowInlineTabs

### PR DESCRIPTION
**Pull request by ckanitz**
Adds documentation for the WordPress.WhiteSpace.DisallowInlineTabs

Related to #1722

### CODE REVIEW

File: WordPress/Docs/WhiteSpace/DisallowInlineTabsStandard.xml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <code_comparison>
+        <code title="Valid: spaces used for inline alignment.">
+        <![CDATA[
+$a = array(
+  'lorem'[space]=> 'ipsum',
+  'dol'[space][space][space]=> 'or',
+);
+        ]]>
+        </code>
+		<code title="Invalid: tabs used for inline alignment.">
```
Comment (GaryJones) [inline WordPress/Docs/WhiteSpace/DisallowInlineTabsStandard.xml]: This line is indented with tabs.

File: WordPress/Docs/WhiteSpace/DisallowInlineTabsStandard.xml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        <![CDATA[
+$a = array(
+  'lorem'[tab]=> 'ipsum',
+  'dol'[tab]=> 'or',
+);
+        ]]>
+        </code>
+	</code_comparison>
```
Comment (GaryJones) [inline WordPress/Docs/WhiteSpace/DisallowInlineTabsStandard.xml]: This line is indented with a tab.

File: WordPress/Docs/WhiteSpace/DisallowInlineTabsStandard.xml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    <![CDATA[
+    Use spaces for inline alignments.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: spaces used for inline alignment.">
+        <![CDATA[
+$a = array(
+  'lorem'[space]=> 'ipsum',
+  'dol'[space][space][space]=> 'or',
```
Comment (GaryJones) [inline WordPress/Docs/WhiteSpace/DisallowInlineTabsStandard.xml]: Each of these array items (4x) should be indented by 4 spaces please.

File: WordPress/Docs/WhiteSpace/DisallowInlineTabsStandard.xml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+<documentation title="Disallow Inline Tabs">
+    <standard>
+    <![CDATA[
+    Use spaces for inline alignments.
```
Comment (jrfnl) [inline WordPress/Docs/WhiteSpace/DisallowInlineTabsStandard.xml]:
```suggestion
    Use spaces for inline alignment.
```

File: WordPress/Docs/WhiteSpace/DisallowInlineTabsStandard.xml [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: spaces used for inline alignment.">
+        <![CDATA[
+$a = array(
+    'a'<em>[space]</em>=> 'lor',
```
Comment (jrfnl) [inline WordPress/Docs/WhiteSpace/DisallowInlineTabsStandard.xml]: May I suggest making this array key three characters long ?
That way the double arrows with the spacing used would be aligned (in real code), while with the current code example people may wonder why the alignment on the second line is done in the first place and why it is off by two.

```suggestion
    'abc'<em>[space]</em>=> 'lor',
```

**Comment by ckanitz** [comment]
hi @jrfnl I commited the changes as requested :)

File: WordPress/Docs/WhiteSpace/DisallowInlineTabsStandard.xml (hunk 5 above)
Comment (jrfnl) [reply WordPress/Docs/WhiteSpace/DisallowInlineTabsStandard.xml]: Sorry to be a pain, but this has now been fixed solely for the "Invalid" example, not (yet) for the "Valid" one :pensive:

Comment (ckanitz) [reply WordPress/Docs/WhiteSpace/DisallowInlineTabsStandard.xml]: sorry! my bad :/
I pushed an update

**Comment by jrfnl** [comment]
@GaryJones Did you still want to have a second look or shall I _Squash/Merge_ this ?

**Pull request merged**